docs/fips_support.md

```diff
@@ -0,0 +1,45 @@
+# Rancher Kubernetes Enginer 2
+
+## Support for FIPS Cryptography
+
+Starting with Rancher Kubernetes Enginer version 2 (RKE2), you can install a cluster that uses FIPS validated libraries. 
```

Comment:
nit: typo in Enginer --> Engine

Reply:
Resolved. Will push all changes at once. Thanks!